packages/react-langgraph/src/useLangGraphRuntime.ts

```
@@ -25,6 +26,37 @@ const getPendingToolCalls = (messages: LangChainMessage[]) => {
+      case "audio":
+        throw new Error("Audio appends are not supported yet.");
+      case "tool-call":
+        throw new Error("Tool call appends are not supported yet.");
+
+      default:
+        const _exhaustiveCheck: never = type;
+        throw new Error(`Unknown content part type: ${_exhaustiveCheck}`);
+    }
+  });
+
+  if (content.length === 1 && content[0]!.type === "text") {
```

Biome

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)